test/binder/binder_error_test.cpp

```diff
@@ -489,3 +489,16 @@ TEST_F(BinderErrorTest, InvalidFixedListSize) {
     auto input = "create node table test1(ID INT64, marks INT64[512], PRIMARY KEY(ID))";
     ASSERT_STREQ(expectedException.c_str(), getBindingError(input).c_str());
 }
+
+TEST_F(BinderErrorTest, MissingStructFieldType) {
```

Comment:
If you are changing grammar, don't forget to remove these 2 tests in your next PR